lib/jackal-kitchen/tester.rb

```diff
@@ -102,8 +114,8 @@ def write_netrc
         begin
           gh_token = config.fetch(:github, :access_token,
                                   app_config.get(:github, :access_token))
-          git_host = config.fetch(:github, :uri,
-                                  app_config.get(:github, :uri))
+          git_host = config.fetch(config.fetch(:github, :uri,
```

Comment:
as an interesting and related fact, the more you're nested in a .fetch method, the slower spacetime appears to the observer